packages/charts/src/components/BarChart/BarChart.tsx

```diff
@@ -219,6 +224,7 @@ const BarChart: FC<BarChartProps> = forwardRef((props: BarChartProps, ref: Ref<H
                 yAxisId={index}
                 width={width[index]}
                 allowDuplicatedCategory={index === 0}
+                orientation={useIsRTL() ? 'right' : 'left'}
```

Comment:
please don't call hooks inside props. extract it to `const isRTL = useIsRTL()` somewhere before the return and just use the value here.